tests/test_files/bundles/verification/crdmissingkindfield.invalid.bundle.yaml

```diff
@@ -0,0 +1,301 @@
+# CRD.spec.names.kind in one of the CRDs is missing in this file
```

Comment:
There have been a number of large yaml files for tests added recently. Should we create some smaller yamls templates that could be modified to test specific errors? @kevinrizza @jeremylinlin

Reply:
@awgreene my preference is a separate yaml for each individual negative test. is there a reason to make them smaller?

Reply:
I would prefer that we have a `tests/templates` dir that contains a few "perfect" CSVs/CRDS/Packages which are copied to a dict and edited to contain the errors that the tests are specifically looking for.

If we continue to use a number of different CSVs, The tests we write will become increasingly polluted as the validation tests are added. If the tests aren't polluted, that means the CSVs will be updated to account for the validation changes. In this case, a user may have to update a number of CSVs to introduce a new validation test. This becomes increasingly difficult to maintain as more tests are added.

Reply:
@awgreene Do you mean we should only load from the "perfect" CSV, programmatically edit the content str after reading the file to introduce the error for the specific test case?

@kevinrizza Thoughts?

Reply:
@awgreene @jeremylinlin 

While I can appreciate the desire to avoid having to edit multiple files, I disagree with the notion that we should write complex code that reads a template file and then modifies it inline (especially for yaml as complex as `csv` files). If that code is anything more than completely trivial, then it becomes necessary to test the test code as well. Which, to me, entirely defeats the purpose of unit tests. It may be slightly time consuming to deal with, but it is significantly simpler in complexity to update a bunch of yaml files with a new field if a verification test is added.